dev/README.md

```diff
@@ -52,22 +50,13 @@ Check the [localconfig-example](bin/localconfig-example) file for an example of
 
 A remote Managed Kafka instance is required for the internal communication between components of the system.
 
-The [kafka-setup.sh](bin/kafka-setup.sh) script takes care of configuring it for you.
-It is **idempotent**, so it can be run whenever you want to make sure the Kafka cluster is configured properly.
+The development team is asked to use the *shared kafka instance* we have deployed under our organization. with the following with the service accounts you can download.
 
-Just run it without arguments:
-
-```bash
-./dev/bin/kafka-setup.sh
-```
-
-**IMPORTANT #1:** the script will perform `rhoas login` for you. Follow the instructions and **log in with your Red Hat account**.
-
-**IMPORTANT #2:** Managed Kafka test instances expire after 48 hours, so you will need to rerun this script at least every 48 hours.
+**Follow the instructions [here](https://docs.google.com/document/d/1fMnHUmGnO-GZuY2BuEe02_prJs3_7QZdSZ5-SfOJ_Yk) to setup the service accounts (you need to be part of the `rhose` google group to access the document)**.
```

Comment:
I checked the document and what about using directly in the doc the `echo` command like
```bash
echo "{ 
    \"this\" : \"is\",
    \"my\" : \"account\"
}" >> dev/bin/credentials/rhose-local-development.json 
```

In this way we avoid to depends on a personal Gist to update

Reply:
done